src/sentry/replays/consumers/recording.py

```diff
@@ -15,6 +15,8 @@
 from sentry_sdk.tracing import Span
 
 from sentry.replays.usecases.ingest import ingest_recording
+from sentry.snuba.utils import initialize_consumer_state
```

Comment:
should not be necessary to use this function

Reply:
Should not or is not?  I'd like to be sure :D

Reply:
Not needed.